pkg/api/latest/latest.go

```diff
@@ -67,6 +67,7 @@ var userResources = []string{"rc", "svc", "pods", "pvc"}
 func init() {
 	// Use the first API version in the list of registered versions as the latest.
 	Version = registered.RegisteredVersions[0]
+	OldestVersion = registered.RegisteredVersions[len(registered.RegisteredVersions)-1]
```

Comment:
This looks wrong-- aren't versions registered in init() functions? if so, the order is arbitrary?